### DISCUSSION TITLE
How to remove/hide the text from ui.linear_progress()?

### DISCUSSION THREAD

**Discussion by smojef** [zauberzeug/nicegui#327] [Q&A] [answered]
How can we display a progress bar without the text (number) when bound to a float value?

**Comment by rodja** [comment, marked as answer]
Set the `show_value` parameter to `False`: https://nicegui.io/reference#linear_progress

**Reply by smojef** [reply]
Thanks!  I was digging through the properties of QLinearProgress, but I missed the obvious ;-)
